pipeline_dp/accumulator.py

```diff
@@ -3,13 +3,29 @@
 import pickle
 from dataclasses import dataclass
 from functools import reduce
+from dataclasses import dataclass
+import pipeline_dp
+
+
+@dataclass
+class AccumulatorParams:
+    accumulator_type: type = None
```

Comment:
Nit: no need in =None for either of those fields (those are not optional parameters)

Reply:
Done!